### PR TITLE
Switch search to react-transit

### PR DESCRIPTION
**Pull request by friedjoff**
- [x] create new react-transit release and update trafimage-maps

### CODE REVIEW

**Comment by friedjoff** [comment]
@lb1c is it possible to make hound less noisy? 99% of this stuff is already covered by ESLint and Prettier which runs in our GitHub action CI.

**Comment by lb1c** [comment]
`hound` was just a test. but since it's more disturbing i'll deactivate it for now

**Comment by friedjoff** [comment]
> `hound` was just a test. but since it's more disturbing i'll deactivate it for now

Thanks! In general I like the idea, but maybe we should use more specific apps like the ImgBot.

**Comment by edellucien** [comment]
We should ensure this MR is a WIP, until we correct the react-transit Search component.
The search functionality isn't quiet prefect yet.

File: src/components/Search/Search.js [hunk 1]
```diff
@@ -1,147 +1,103 @@
...
-    };
-  }, []);
+function Search({ apiKey }) {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const { layerService, map, searchEngines } = useSelector(state => ({
+    layerService: state.app.layerService,
+    map: state.app.map,
+    searchEngines: state.app.activeTopic.searchEngines,
+  }));
```
Comment (oterral) [inline src/components/Search/Search.js]: this generates always a rerender of the component . I will separate in 3 useSelector instead.

**Comment by oterral** [comment]
Is this still actual ? @mhaertwig ?

**Pull request closed without merging**